qemu/patches/4.1.x/0001-9p-removing-coroutines-of-9p-to-increase-the-I-O-per.patch

```diff
@@ -0,0 +1,98 @@
+From 894a8ae5524fff4ad9d3551c515788c5650e1fc7 Mon Sep 17 00:00:00 2001
+From: Yang Zhong <yang.zhong@intel.com>
+Date: Wed, 28 Mar 2018 20:14:53 +0800
+Subject: [PATCH 1/2] 9p: removing coroutines of 9p to increase the I/O
```

Comment:
It might be worth adding details to the commit of:

- Where this patch originally came from (canonical URL)?
- Why we're not including the 2nd patch?

Reply:
Fixed, thanks